packages/components/visually-hidden/tests/visually-hidden.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from "@yamada-ui/test"
+import { VisuallyHidden } from "../src"
+
+describe("<VisuallyHidden />", () => {
+  test("VisuallyHidden renders correctly", async () => {
+    render(<VisuallyHidden>Test</VisuallyHidden>)
+    const el = screen.getByText("Test")
+    expect(el.textContent).toBe("Test")
+  })
+
+  test("VisuallyHidden is visually hidden", async () => {
+    render(<VisuallyHidden>Test</VisuallyHidden>)
+    const style = window.getComputedStyle(screen.getByText("Test"))
```

Comment:
Wouldn't `toHaveStyle` be better?

https://github.com/testing-library/jest-dom?tab=readme-ov-file#tohavestyle

Reply:
This PR is to raise the version and does not require a review. If there is an improvement, please create another issue and discuss it.